### DISCUSSION TITLE
Unable to install modulus on Linux with Python 3.8

### DISCUSSION THREAD

**Discussion by AAKHASH17** [NVIDIA/physicsnemo#730] [General]
Can someone help me to install modulus on Linux with python 3.8 and Nvidia RTX 3050 GPU?

**Comment by AumGupta** [comment]
Hey @AAKHASH17 ,

NVIDIA has given a pretty impressive installation doc:
https://docs.nvidia.com/deeplearning/modulus/getting-started/index.html

Please follow this.

**Comment by AAKHASH17** [comment]
I follow the exact same steps for doing a pip install, but I get some
random errors saying the build failed.

**Reply by AumGupta** [reply]
Please provide the exact `errors` that you are getting.
